Last-Update: 2014-11-15
Forwarded: yes
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Bug-Debian: http://bugs.debian.org/769585
Description: disable CD-quality check, needed to process 24bit 96kHz files.

--- a/src/core_mode.c
+++ b/src/core_mode.c
@@ -344,10 +344,10 @@
 
   *colon = 0;
   *dot = 0;
 
-  if (PROB_NOT_CD(info))
-    st_error("m:ss.ff format can only be used with CD-quality files");
+//  if (PROB_NOT_CD(info))
+//    st_error("m:ss.ff format can only be used with CD-quality files");
 
   min = atoi((const char *)buf);
   sec = atoi((const char *)(colon+1));
   frames = atoi((const char *)(dot+1));
